src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: #fff;
```

Comment:
Consider using SCSS variables for colors, font sizes, and other repeated values to enhance maintainability and readability.